scripts/dotnet-cli-build/TestTargets.cs

```diff
@@ -178,7 +165,7 @@ public static BuildTargetResult BuildTestAssetPackages(BuildTargetContext c)
                 {
                     "netstandard1.5",
                     "netstandard1.3",
-                    "netstandardapp1.5"
+                    "netcoreapp1.0"
```

Comment:
I agree with the intent, but would prefer to see a refactoring. Comments are impossible to keep up to date.